From c8b904ac21718224922633ed7578d9fc02a75c0a Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Thu, 24 Sep 2020 06:50:55 -0700
Subject: [PATCH 1658/1851] dmaengine: xilinx: frmbuf: Fix warnings for UNINIT

commit 2c112bc0803f4e1df13bee6f343418e28edc159e from
https://github.com/Xilinx/linux-xlnx.git

This patch fix the warnings for the UNINIT type.

The actual warnings are:
For xilinx_frmbuf_chan_probe():

Event var_decl: Declaring variable "dma_addr_size" without
initializer.
Event uninit_use: Using uninitialized value "dma_addr_size".

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/dma/xilinx/xilinx_frmbuf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dma/xilinx/xilinx_frmbuf.c b/drivers/dma/xilinx/xilinx_frmbuf.c
index 5ab1a1e66132..026d593848df 100644
--- a/drivers/dma/xilinx/xilinx_frmbuf.c
+++ b/drivers/dma/xilinx/xilinx_frmbuf.c
@@ -1416,7 +1416,7 @@ static int xilinx_frmbuf_chan_probe(struct xilinx_frmbuf_device *xdev,
 {
 	struct xilinx_frmbuf_chan *chan;
 	int err;
-	u32 dma_addr_size;
+	u32 dma_addr_size = 0;
 
 	chan = &xdev->chan;
 
-- 
2.31.1

